### ISSUE TITLE
 Bot is broken ?

### ISSUE THREAD

**Issue by nil0x42** [bp0lr/dmut-resolvers#5]
Last update is 2 weeks old.
Bot might be broken

**Comment by bp0lr** [comment]
Yeah, the vps used to run the process to check the validators was shut down.
I will try to get a new one.
